### ISSUE TITLE
[Bug] Website icons not in sync with release tag

### ISSUE THREAD

**Issue by geoffreymcgill** [primer/octicons#738]
The `v16.3.0` icons published on the [website](https://octicons-primer.vercel.app/octicons/upload-24) are not in sync with the `v16.3.0` [release](https://github.com/primer/octicons/releases/tag/v16.3.0) tag.

<img width="382" alt="Screen Shot 2022-02-05 at 1 45 52 PM" src="https://user-images.githubusercontent.com/62210/152658923-18ff3816-1b5a-4a1c-931f-c2e305b73a36.png">

For example, see the [`upload`](https://octicons-primer.vercel.app/octicons/upload-24) icon on the website:

<img width="726" alt="Screen Shot 2022-02-05 at 1 51 30 PM" src="https://user-images.githubusercontent.com/62210/152658605-dd4860a2-2568-44a7-bd1a-a50c28bab618.png">

... and here is the [`upload`](https://github.com/primer/octicons/blob/main/icons/upload-24.svg) icon in the `main` branch:

<img width="43" alt="Screen Shot 2022-02-05 at 2 02 46 PM" src="https://user-images.githubusercontent.com/62210/152658976-b3921460-0fbc-4bef-94a2-68aff5b6b28f.png">

... vs the [`upload`](https://github.com/primer/octicons/blob/v16.3.0/icons/upload-24.svg) icon in the `v16.3.0` release tag:

<img width="48" alt="Screen Shot 2022-02-05 at 1 52 15 PM" src="https://user-images.githubusercontent.com/62210/152658652-bf171568-5151-4b45-b5ed-e38d0fccf6eb.png">

**Comment by ashygee** [comment]
Hi @geoffreymcgill, because we are using [changesets](https://github.com/primer/octicons/blob/main/CONTRIBUTING.md#releasing-changes) to cut releases we have PRs automatically created for our release tracking. The updated `upload` icon will be in 16.3.1 as seen in this PR. https://github.com/primer/octicons/pull/735

**Comment by fabriciomurta** [comment]
Ashygee, I think you have the [website publish workflow](https://github.com/primer/octicons/blob/main/.github/workflows/deploy_production.yml#L2-L5) set for every push instead of every "release publish". So once the pull request for the upload icon was merged, the new icon reflected in the website while reporting version 16.3.0.

Now that release 16.3.1 occurred, the website is up-to-date (has 16.3.1 changes and report version 16.3.1). So I guess it is pushing a "latest" version before the version number is actually switched and, when the version string is bumped (with the release) the "push" with the version changes trigger again deployment and the site stays in sync with the reported version... until next pull request is merged. :)

In other words, the way the deploy is set up now will reflect current `main` in the website while reporting the latest full release version string, which made us some confusion about the upload icon and version it was included in.

**Comment by geoffreymcgill** [comment]
The original issue will still happen and is a valid issue. Please consider re-opening. 

At the time, the website was sync'd with `main` (the next release?) and not the latest Release tag.
